src/content/blog/community-day-2023.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: "Building a community of builders"
+description: "Today, we launch our newest initiative to recognize people in our community who go above and beyond, including our very own maintainers."
+publishDate: "December 7, 2023"
+coverImage: "/src/content/blog/_images/community-day-2023/header.webp"
+socialImage: "/src/content/blog/_images/community-day-2023/og.webp"
+lang: "en"
+authors:
+  - sarah
+  - elian
+---
+
+import CommunityProject from '../../components/CommunityProject.astro'
+
+Astro continues to evolve how it works with and for its community. 
+
+Our community awards allow us to give back to those who volunteer their time submitting code and bug fixes, being welcoming and helpful in support threads, translating the docs, and enriching everyone's Astro experience.
+
+Previously a quarterly event, Astro recently committed to issuing these awards on a monthly basis. We also changed from peer-nominated awards to recipients chosen by our core maintainers.
+
+This might sound less democratic, but we think of it like open-source maintainer gratitude journaling: **we ensure that our core maintainers are actively reflecting, and on the lookout for the next crop of up-and-comers**. We are proud to say that we dedicate time to our open-source community's health, growth, and sustainability.
```

Comment:
comma nit
```suggestion
This might sound less democratic, but we think of it like open-source maintainer gratitude journaling: **we ensure that our core maintainers are actively reflecting and on the lookout for the next crop of up-and-comers**. We are proud to say that we dedicate time to our open-source community's health, growth, and sustainability.
```